R/zzz.R

```diff
@@ -14,5 +14,6 @@
   }
 
   .initPackage()
+  ospsuiteEnv$illegalCharacters <- .getIllegalCharacters()
```

Comment:
@Yuri05  could you re-approve. I had to move the call to .NET to here because otherwise the checks didn't pass (I still don't understand how R handles some things...)

Reply:
Approved.
R = <img width="24" src="https://github.com/user-attachments/assets/ff6beab4-8067-44bd-b23f-cb754e59116e">